src/components/Email/Email.scss

```diff
@@ -43,7 +67,11 @@
         &__form{
             max-height: 600px;
             opacity: 1;
-            transition: all .2s ease;
+            transition: all .5s ease;
+
+            @include respond(phone){
+                margin-left: -25%;
```

Comment:
This probably means that you should organize your grid differently, instead of relying on negative margins

Reply:
I got rid of negative margins. I positioned elements differently.